src/guide/essentials/component-basics.md

```diff
@@ -6,7 +6,7 @@ Components allow us to split the UI into independent and reusable pieces, and th
 
 <!-- https://www.figma.com/file/qa7WHDQRWuEZNRs7iZRZSI/components -->
```

Comment:
I think this should be removed as well

Reply:
Nahh.... it's a quick path for us to get the Figma draft. Maybe readers wants to fork and re-edit it for their study notes ?

Reply:
Alright